ocaml/xapi/xapi_vbd_helpers.ml

```diff
@@ -169,12 +169,12 @@ let valid_operations ~expensive_sharing_checks ~__context record _ref' : table =
     if not vdi_record.Db_actions.vDI_managed
     then set_errors Api_errors.vdi_not_managed [ _ref ] all_ops;
 
-    let vdi_operations_besides_copy =
+    let operation_cannot_be_performed_live=
       List.exists
-        (fun (_, operation) -> operation <> `copy)
```

Comment:
I like this but then I would go all the way: I'd prefer to list all the legal operations as well. This would raise a warning if we add new operations in the future and we do not update this function.